x-pack/plugins/monitoring/server/lib/metrics/beats/metrics.js

```diff
@@ -248,5 +248,15 @@ export const metrics = {
     format: LARGE_FLOAT,
     metricAgg: 'max',
     units: ''
+  }),
+
+  beat_handles_open: new BeatsMetric({
+    field: 'beats_stats.metrics.beat.handles.open',
+    label: 'Open Handles',
+    title: 'Open Handles',
+    description: 'Count of open file handlers',
+    format: SMALL_FLOAT,
```

Comment:
Why do we use `FLOAT` here? open file handlers should be an INT / LONG?

Reply:
Good question. [That's not an option here](https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/common/formatting.js#L10) @pickypg @tsullivan  for clarity

Reply:
We should add it.